beetmoverscript/templates/fennecx86_nightly_en_us_multi_signed.yml

```diff
@@ -6,12 +6,10 @@ mapping:
   {{ locale }}:
   {% set locale_prefix = 'en-US/' if locale == 'en-US' else '' %}
   {% set update_balrog_manifest = True if locale == 'multi' else False %}
-    apk:
-      artifact: {{ locale_prefix }}target.apk
+    target.apk:
       s3_key: {{ locale_prefix  }}fennec-{{ version }}.{{ locale }}.android-i386.apk
       update_balrog_manifest: {{ update_balrog_manifest }}
-    # Bug 1316124 - TODO
-    #checksums:
-      #artifact: {{ locale_prefix }}target.checksums
+    # Bug 1316124 - TODO figure out checksums
+    #target.checksums:
```

Comment:
As above, unrelated to the code change, but a reminder for us all to clarify this before Tier-1 (bug 1316124)